<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ExportXmlViewerClass</class>
 <widget class="QMainWindow" name="ExportXmlViewerClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>818</width>
    <height>659</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>ExportXmlViewer</string>
  </property>
  <widget class="QWidget" name="centralWidget"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>818</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_File">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="action_Open_Xml_File"/>
    <addaction name="separator"/>
    <addaction name="action_Quit"/>
   </widget>
   <widget class="QMenu" name="menu_View">
    <property name="title">
     <string>&amp;View</string>
    </property>
    <addaction name="action_Show_Wave_Info"/>
    <addaction name="separator"/>
    <addaction name="action_Draw_Waveforms"/>
    <addaction name="action_Draw_MedianBeatTemplates"/>
    <addaction name="separator"/>
    <addaction name="action_List_Observations"/>
    <addaction name="action_Patient_Information"/>
   </widget>
   <widget class="QMenu" name="menu_Setting">
    <property name="title">
     <string>&amp;Settings</string>
    </property>
    <addaction name="action_Display_Settings"/>
   </widget>
   <widget class="QMenu" name="menu_Help">
    <property name="title">
     <string>&amp;Help</string>
    </property>
    <addaction name="action_About"/>
   </widget>
   <addaction name="menu_File"/>
   <addaction name="menu_View"/>
   <addaction name="menu_Setting"/>
   <addaction name="menu_Help"/>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <action name="action_Open_Xml_File">
   <property name="icon">
    <iconset>
     <normalon>:/exportxmlviewer/open.png</normalon>
    </iconset>
   </property>
   <property name="text">
    <string>&amp;Open XML File</string>
   </property>
  </action>
  <action name="action_Quit">
   <property name="text">
    <string>&amp;Quit</string>
   </property>
  </action>
  <action name="action_Show_Wave_Info">
   <property name="icon">
    <iconset>
     <normalon>:/exportxmlviewer/information.png</normalon>
    </iconset>
   </property>
   <property name="text">
    <string>&amp;Show Wave Information</string>
   </property>
  </action>
  <action name="action_Draw_Waveforms">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="qexportxmlviewer.qrc">
     <normaloff>:/exportxmlviewer/waveforms.png</normaloff>:/exportxmlviewer/waveforms.png</iconset>
   </property>
   <property name="text">
    <string>Draw &amp;Waveforms</string>
   </property>
   <property name="iconText">
    <string>Draw Waveforms</string>
   </property>
  </action>
  <action name="action_Draw_MedianBeatTemplates">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="qexportxmlviewer.qrc">
     <normaloff>:/exportxmlviewer/medianbeattemplates.png</normaloff>:/exportxmlviewer/medianbeattemplates.png</iconset>
   </property>
   <property name="text">
    <string>Draw &amp;MedianBeatTemplates</string>
   </property>
   <property name="iconText">
    <string>Draw MedianBeatTemplates</string>
   </property>
  </action>
  <action name="action_List_Observations">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="qexportxmlviewer.qrc">
     <normaloff>:/exportxmlviewer/parameters.png</normaloff>:/exportxmlviewer/parameters.png</iconset>
   </property>
   <property name="text">
    <string>&amp;List Observation</string>
   </property>
   <property name="iconText">
    <string>List Observations</string>
   </property>
  </action>
  <action name="action_Patient_Information">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="qexportxmlviewer.qrc">
     <normaloff>:/exportxmlviewer/patient.png</normaloff>:/exportxmlviewer/patient.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Patient Information</string>
   </property>
   <property name="iconText">
    <string>Patient Information</string>
   </property>
  </action>
  <action name="action_Display_Settings">
   <property name="icon">
    <iconset resource="qexportxmlviewer.qrc">
     <normaloff>:/exportxmlviewer/settings.png</normaloff>:/exportxmlviewer/settings.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Display Settings</string>
   </property>
  </action>
  <action name="action_About">
   <property name="icon">
    <iconset resource="qexportxmlviewer.qrc">
     <normaloff>:/exportxmlviewer/help.png</normaloff>:/exportxmlviewer/help.png</iconset>
   </property>
   <property name="text">
    <string>&amp;About ...</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="qexportxmlviewer.qrc"/>
 </resources>
 <connections/>
</ui>
